### PR TITLE
added translation in portuguese

### PR DESCRIPTION
**Pull request by MarianaPinto17**
This pull request aims to enhance the functionality and accessibility of the Django package by adding Portuguese translation support.

The changes proposed in this pull request primarily involve the addition of translation files and related configuration settings. Here is an overview of the modifications:

Locale Files:
    - Added a new directory, "pt," within the "locale" folder.
    - Created the "django.po" file in the "pt" directory to store the Portuguese translation strings.
    - Added verbose_name in some fields and associated migrations

### CODE REVIEW

File: import_export_celery/admin.py [hunk 1]
```diff
@@ -9,6 +9,7 @@
 
 
 class JobWithStatusMixin:
+    @admin.display(description=_("Job status info"))
```
Comment (timthelion) [inline import_export_celery/admin.py:12]: Why was this removed?

**Comment by timthelion** [comment]
Awesome, thanks for the work. I just had one question which I left in the review.

File: import_export_celery/admin.py (hunk 1 above)
Comment (MarianaPinto17) [reply import_export_celery/admin.py:12]: removed? I think you misread. I added this line with the wrapper to provide a verbose name for the custom admin propery of job_status_info :)

Comment (diogosilva30) [reply import_export_celery/admin.py:12]: Any updates on this @timthelion ? I would also benefit from such translations

Comment (timthelion) [reply import_export_celery/admin.py:12]: Aha, sorry, I did misread!

**Review by timthelion — APPROVED** [review]

**Comment by MarianaPinto17** [comment]
Hi, @timthelion! Can you please release a new version with these alterations? Or when are you excepting to make a new release? Thanks!

**Comment by timthelion** [comment]
Just pushed a new release
